_vendor/github.com/inspec/inspec-aws/docs-chef-io/static/release-notes/inspec-aws/release-dates.json

```
@@ -1,4 +1,5 @@
 [
   "2021-10-01",
   "2021-10-13"
```

Software is great when it works and annoying when it doesn't. Now I have to fix this missing comma issue in inspec-aws.
```suggestion
  "2021-10-13",
```